.circleci/config.yml

```diff
@@ -321,7 +321,8 @@ jobs:
            environment:
              BUILD: "cabal-v2"
              CABAL_BUILD_OPTIONS: "--flag fusion-plugin"
-             DISABLE_SDIST_BUILD: "no"
+             DISABLE_SDIST_BUILD: "yes"
+             CABAL_PROJECT: "cabal.project"
```

Comment:
Are you going to commit these?

Reply:
Yes, I'll revert them later.

Reply:
Before releasing. Or should we keep the CIs failing?

Reply:
Or keep this PR open till we upload streamly?